### ISSUE TITLE
spigot 1.13 + fastlogin 

### ISSUE THREAD

**Issue by minecraft7net** [games647/FastLogin#216]
[01:36:16] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-ed1cec9-1c7adf7 (MC: 1.13) (Implementing API version 1.13-R0.1-SNAPSHOT)

[01:36:26] [Server thread/ERROR]: Error occurred while enabling FastLogin v1.11-SNAPSHOT-c172b1e (Is it up to date?)
java.lang.IllegalArgumentException: Channel must contain : separator
        at org.bukkit.plugin.messaging.StandardMessenger.validateAndCorrectChannel(StandardMessenger.java:474) ~[spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at org.bukkit.plugin.messaging.StandardMessenger.registerIncomingPluginChannel(StandardMessenger.java:209) ~[spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at com.github.games647.fastlogin.bukkit.FastLoginBukkit.onEnable(FastLoginBukkit.java:70) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:339) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugin(CraftServer.java:424) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugins(CraftServer.java:338) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at net.minecraft.server.v1_13_R1.DedicatedServer.init(DedicatedServer.java:214) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:686) [spigot-1.13-b1658.jar:git-Spigot-ed1cec9-1c7adf7]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_152]

**Comment by k-jiang** [comment]
Players are prevented from joining a server, with this exception thrown:
```
[04:47:07 INFO]: This server is running CraftBukkit version git-Spigot-a85f7ec-5870145 (MC: 1.13) (Implementing API version 1.13-R0.1-SNAPSHOT)
[04:47:07 INFO]: Checking version, please wait...
[04:47:09 INFO]: You are running the latest version
[04:47:19 INFO]: UUID of player k_jiang is c726210d-3489-4e00-9aca-d3f427d7b294
[04:47:20 INFO]: k_jiang[/108.168.47.203:34602] logged in with entity id 66 at ([world]107.68306746775954, 68.0, 67.47650343156607)
[04:47:25 ERROR]: Couldn't register custom payload
java.lang.IllegalArgumentException: Channel must be entirely lowercase
        at org.bukkit.plugin.messaging.StandardMessenger.validateAndCorrectChannel(StandardMessenger.java:478) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at org.bukkit.craftbukkit.v1_13_R1.entity.CraftPlayer.addChannel(CraftPlayer.java:1203) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.PlayerConnection.a(PlayerConnection.java:2434) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.PacketPlayInCustomPayload.a(SourceFile:45) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.PacketPlayInCustomPayload.a(SourceFile:9) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.PlayerConnectionUtils.a(SourceFile:10) ~[spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_181]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_181]
        at net.minecraft.server.v1_13_R1.SystemUtils.a(SourceFile:198) [spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.MinecraftServer.w(MinecraftServer.java:884) [spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.DedicatedServer.w(DedicatedServer.java:411) [spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.MinecraftServer.v(MinecraftServer.java:819) [spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:717) [spigot-vol5.jar:git-Spigot-a85f7ec-5870145]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
[04:47:25 INFO]: k_jiang lost connection: Invalid payload REGISTER!
[04:47:25 INFO]: k_jiang left the game
[04:47:25 WARN]: handleDisconnection() called twice
```
Tested with:
[FastLogin build 839](https://ci.codemc.org/job/Games647/job/FastLogin/839/)
[Waterfall build 210](https://destroystokyo.com/ci/job/Waterfall/210/)
Spigot build 1737
Client version: 1.13 vanailia

**Comment by games647** [comment]
@k-jiang  Thanks for posting that exception. It should be fixed now in 542aabad73c2f45c0ab21a3190716a131b3a024d
